src/Microsoft.AspNet.Mvc.ViewFeatures/Resources.resx

```diff
@@ -259,4 +259,7 @@
   <data name="RemoteAttribute_RemoteValidationFailed" xml:space="preserve">
     <value>'{0}' is invalid.</value>
   </data>
+  <data name="HtmlGenerator_FieldNameCannotBeNullOrEmpty" xml:space="preserve">
+    <value>The field name for Html field cannot be Null or Empty. Please provide a non-Empty field name OR use 'EditorFor' as a workaround with parameter name=null.</value>
```

Comment:
I meant the method that exists on the generic interface.

Looks like `nameof` doesn't like the open generic type; use `nameof(IHtmlHelper<object>.EditorFor)`.